src/library_sdl.js

```diff
@@ -16,18 +16,8 @@
 //                   or otherwise).
 
 var LibrarySDL = {
-  $SDL_unicode__docs: '/** @suppress{missingProperties} */',
-  $SDL_unicode: () => SDL.unicode,
-
-  $SDL_ttfContext__docs: '/** @suppress{missingProperties} */',
-  $SDL_ttfContext: () => SDL.ttfContext,
-
-  $SDL_audio__docs: '/** @suppress{missingProperties} */',
-  $SDL_audio: () => SDL.audio,
```

Comment:
I guess there is no chance these are used by any other port as this is the SDL1 port which should not interact with `SDL2_*`?

Reply:
Yes, the chances of these being used external are absolutely tiny I think.